webpack.react.dev.js

```diff
@@ -82,10 +86,26 @@ module.exports = {
                 // Include ts, tsx, js, and jsx files.
                 test: /\.(ts|js)x?$/,
                 exclude: [/node_modules/, /\.test\.ts$/, /\.spec\.ts$/],
-                use: [{ loader: 'ts-loader', options: { transpileOnly: true, onlyCompileBundledFiles: true } }],
+                use: [
+                    {
+                        loader: '@compiled/webpack-loader',
+                        options: {
+                            transformerBabelPlugins: ['@atlaskit/tokens/babel-plugin'],
+                            extract: true,
+                            inlineCss: true,
+                        },
+                    },
+                    { loader: 'babel-loader' },
```

Comment:
Why add the babel-loader? This will hurt build performance so should be avoided if possible